internal/transform/operation.go

```diff
@@ -279,6 +279,7 @@ func addAssetDetailsToOperationDetails(result map[string]interface{}, asset xdr.
 
 	result[prefix+"asset_code"] = code
 	result[prefix+"asset_issuer"] = issuer
+	result[prefix+"asset_id"] = FarmHashAsset(code, issuer, assetType)
```

Comment:
Is this the only `details` field that needs an asset id? What about trade operations or liquidity pool operations where the asset fields are `selling_asset_code` or `asset_a_code` etc?

Curious if this field is the biggest impact or if we need the other asset fields to have the asset id as well.

Reply:
Nevermind, I see this applies for all asset fields